api/add.go

```diff
@@ -13,10 +15,10 @@ var DefaultShardSize = uint64(100 * 1024 * 1024) // 100 MB
 // AddedOutput carries information for displaying the standard ipfs output
 // indicating a node of a file has been added.
 type AddedOutput struct {
-	Name  string `json:"name"`
-	Cid   string `json:"cid,omitempty"`
-	Bytes uint64 `json:"bytes,omitempty"`
-	Size  uint64 `json:"size,omitempty"`
+	Name  string  `json:"name" codec:"n,omitempty"`
```

Comment:
You want to omit this from codec, but not from json, why?

Reply:
not sure. I don't think we use the codec anyway. The json always carrying `Name` is probably because that's what ipfs did with the equivalent type. It's nice if things come out of the API with Name and Cid at least, even if name is empty.